containers/Dockerfile-ppc64le

```diff
@@ -0,0 +1,25 @@
+# Dockerfile for kraken
+
+FROM ppc64le/centos:8
```

Comment:
@pravin-dsilva This is great. We might want to add document around using an image built using this dockerfile when running Kraken on Power: https://github.com/cloud-bulldozer/kraken/blob/master/docs/installation.md#run-containerized-version. Thoughts?

Reply:
Sure, I have added some instructions now. Please take a look, Thanks.